docs/quick-reference.adoc

```diff
@@ -1,2 +1,215 @@
 = Sylt Quick Reference
 :favicon: sylt.png
+
+Here you will find most things in Sylt in a concise format.
+
+.Variables
+[source, sylt]
+----
+constant :: 3.14
+variable := []
+two: int : 2
+text: str = "abc"
+text = "A short sentence."  // <1>
+----
+
+<1> A single `=` is used to set the value of a variable after declaration with
+`:=`.
```

Comment:
This reads poorly to me, I don't know...

```suggestion
<1> A single `=` is used to mutate and set the value of a mutable variable.
Mutable variables are declared with `:=`.
```
?

Reply:
Updated the formulation a bit.